tests/ttsim/mettsim/housing_benefits/params.yaml

```
@@ -6,6 +6,8 @@ assistance_rate:
   description:
     de: In Mittelerde passiert immer noch nichts.
     en: Still nothing happens in Middle Earth.
+  unit: Share
```

```suggestion
  unit: null
```

Could be larger than 1